wordpress-mobile/WordPress-iOS
# Stats - Insights - Update Default cards list

Fixes #18042 

This pull request updates the default Insights screen. This also removes the default "Latest Post Summary" card and removes the "Customize" card from `UserDefaults`

**NOTE: this pull request does not remove the Customize card entirely, just from `UserDefaults`. This is currently being addressed in a separate PR**

To test:
Pre-requisite: have a wp.com account with at least one site (better if more)
Test steps:
- Install a previous version than this branch (e.g. `trunk`)
- Choose a site where you have not customized the Insights screen in Stats
- Go to My Site -> Stats -> Insights
- Verify that the default cards are: "Latest Post Summary", "Today", "All Time" and "Follower Totals", plus any pinned cards (e.g. "Grow Audience or "Customize")
- Build/run this branch
- verify that the default cards now are: "Most Popular Time", "All Time", "Today", "Followers", "Comments"
- Repeat the steps with several sites and observe the behavior: previously customized pages should be left as they were

## Regression Notes
1. Potential unintended areas of impact
not really, this PR only reshuffles default cards in Insights

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Test the feature on multiple sites with different scenarios

3. What automated tests I added (or what prevented me from doing so)
None
PR submission checklist:
